src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const getLinkClass = ({ isActive }: { isActive: boolean }) => classNames(
+  'navbar-item',
+  {
+    'has-background-grey-lighter': isActive,
+  },
+);
```

Comment:
You create the same function several times (App.js line 6) , it can be exported as a general helper and reused